tests/qc_checkers/test_percent_q30.py

```diff
@@ -0,0 +1,130 @@
+from collections import namedtuple
+import numpy as np
+import pytest
+
+
+from checkQC.qc_checkers.percent_q30 import percent_q30
+from checkQC.handlers.qc_handler import QCErrorFatal, QCErrorWarning
+
+from tests.test_utils import float_eq
+
+
+@pytest.fixture
+def qc_data_and_exp_val():
+    return (
+        namedtuple("QCData", "sequencing_metrics")(
+            {
+                1:{
+                    "reads": {
+                        1: {"percent_q30": 82.},
```

Comment:
Consequently, you should specify `is_index` everywhere here. This also makes the test data more explicit.